src/components/dataset/show.vue

```diff
@@ -113,14 +113,14 @@ export default {
     const { tabPath, tabClass } = useTabs(datasetPath());
     return {
       project, dataset, ...resourceStates([project, dataset]),
-      projectPath, tabPath, tabClass, canRoute, publishedFormPath
+      datasetPath, projectPath, tabPath, tabClass, canRoute, publishedFormPath
     };
   },
   computed: {
     breadcrumbLinks() {
       return [
-        { text: this.project.nameWithArchived, path: this.projectPath() },
-        { text: this.$t('resource.entities'), path: this.projectPath('entity-lists'), icon: 'icon-database' }
+        { text: this.project.dataExists ? this.project.nameWithArchived : this.$t('resource.project'), path: this.projectPath('entity-lists'), icon: 'icon-archive' },
```

Comment:
I think the project is guaranteed to exist, so you don't need to check `this.project.dataExists`. `<breadcrumbs>` has a `v-if="dataExists"`, and `dataExists` (from `resourceStates()`) includes the project. Definitely OK though to check for project existence if you prefer / if that seems more robust.